specs/core/1_shard-data-chains.md

```diff
@@ -47,14 +47,16 @@ This document describes the shard data layer and the shard fork choice rule in P
 | `BYTES_PER_SHARD_BLOCK_BODY` | `2**14` (= 16,384) |
 | `MAX_SHARD_ATTESTIONS` | `2**4` (= 16) |
 | `PHASE_1_GENESIS_EPOCH` | **TBD** |
-| `PHASE_1_GENESIS_SLOT` | get_epoch_start_slot(PHASE_1_GENESIS_EPOCH) |
+| `PHASE_1_GENESIS_SLOT` | **TBD** |
+| `GENESIS_SHARD_SLOT` | 0 |
 
 ### Time parameters
 
 | Name | Value | Unit | Duration |
 | - | - | :-: | :-: |
 | `CROSSLINK_LOOKBACK` | `2**0` (= 1) | epochs  | 6.2 minutes |
 | `PERSISTENT_COMMITTEE_PERIOD` | `2**11` (= 2,048) | epochs | ~9 days |
+| `SECONDS_PER_SLOT` | `2**1 * 3**1` (= 6) | 6 seconds |
```

Comment:
This is now defined in two different places -- fork choice doc and this shard chain doc.

@JustinDrake would argue that the `assert` using `SECONDS_PER_SLOT` here should instead be in the fork choice to limit the tree under consideration.

Reply:
Yes, but I repeated it for the purposes of having the spec be executable as the fork choice not being executable means that `SECONDS_PER_SLOT ` is not in the scope of spec.py

I agree that it should not necessarily be a condition for block validity at a state-transition level. The reason I like having this check in the fork choice is less for tree limiting and more for having block validity purely dependent on state and the block's contents.